definitions/infra-awsapigatewayresourcewithmetrics/summary_metrics.yml

```
@@ -5,22 +5,22 @@ providerAccountName:
   unit: STRING
 requests:
   query:
-    eventId: entityGuid
-    select: sum(`provider.count.SampleCount`)
-    from: ApiGatewaySample
+    eventId: entity.guid
+    select: sum(aws.apigateway.Count.byResourceWithMetrics)
+    from: Metric
   unit: COUNT
   title: Requests
 errors:
   query:
-    eventId: entityGuid
-    select: sum(`provider.5xxError.Sum`) + sum(`provider.4xxError.Sum`)
-    from: ApiGatewaySample
+    eventId: entity.guid
+    select: sum(aws.apigateway.5XXError.byResourceWithMetrics) + sum(aws.apigateway.4XXError.byResourceWithMetrics)
```

As per the bot complaints:

```suggestion
    select: sum(`aws.apigateway.5XXError.byResourceWithMetrics`) + sum(`aws.apigateway.4XXError.byResourceWithMetrics`)
```